frontend/src/components/CustomButton.tsx

```diff
@@ -24,4 +24,28 @@ const CustomSubmitButton = ({
   );
 };
 
+export const CustomButton = ({
```

Comment:
I think we can rewrite it like this. whenever we create the component. the component itself should have props to accept.
```suggestion
interface PropsToAccept {
pendingStatus: TxStatus;
.....
}

export const CustomButton: React.FC<PropsToAccept> = ({pending..})
```